community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -191,14 +191,27 @@
     @Description( "The number of Cypher query execution plans that are cached." )
     public static Setting<Integer> query_cache_size = setting( "dbms.query_cache_size", INTEGER, "1000", min( 0 ) );
 
-    @Description( "The threshold when a plan is considered stale. If any of the underlying" +
-                  " statistics used to create the plan has changed more than this value, " +
-                  "the plan is considered stale and will be replanned. " +
-                  "A value of 0 means always replan, and 1 means never replan." )
+    @Description( "The threshold when a plan is considered stale. If any of the underlying " +
+                  "statistics used to create the plan have changed more than this value, " +
+                  "the plan will be considered stale and will be replanned. Change is calculated as " +
+                  "abs(a-b)/max(a,b). This means that a value of 0.75 requires the database to approximately " +
```

Comment:
If you want this here I think you need to explain what `a` and `b` are (and they are not always cardinalities, could also be a selectivity)